distribution/changelog.txt

```diff
@@ -1,5 +1,6 @@
 0.3.5+ (in development)
 ------------------------------------------------------------------------
+- Feature: [#15901] [Plugin] Add function "map.getAllEntitiesOnTile"
```

Comment:
Since there is a new release, this has to be rebased and moved up to the current version.